src/main/java/service/parse/DataParserImpl.java

```diff
@@ -0,0 +1,27 @@
+package service.parse;
+
+import model.FruitRecord;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataParserImpl implements DataParser {
```

Comment:
The first line of the class implementation should not be empty. Please remove the empty line to follow the common practices and checklist requirements.